core/modules/filters.js

```diff
@@ -30,7 +30,8 @@ function parseFilterOperation(operators,filterString,p) {
 		operator = {};
 		// Check for an operator prefix
 		if(filterString.charAt(p) === "!") {
-			operator.prefix = filterString.charAt(p++);
+			operator.prefix = filterString.charAt(p++); // remove?
```

Comment:
Yes, but separation of concerns would be better if we fixed the list filter independently of introducing operator.negate. it's about having a readable commit history.